### ISSUE TITLE
Issues I've noticed since the update

### ISSUE THREAD

**Issue by Legitosity** [jweigelt/swbf2admin#17]
RCON has worked like a charm since the update until yesterday when I got a threading error. I've attached a picture of the console message below. A window also pops up saying "SWBFAdmin has stopped working".

The server still disappears from the steam server browser sometimes, but it's not crashing because it's still running on the VPS. The error message I get is a pop up window saying "BattlefrontII.exe has stopped working"

The last thing is I can't figure out is why the stats page no longer works for me, but I still need to do some more testing on this one.

Thanks for your continued work!

**Comment by jweigelt** [comment]
Seems like there is an issue with player statistics for new matches not being stored. Going to investigate..

**Comment by yonilerner** [comment]
@jweigelt In my testing I noticed that if the match ended within 30 seconds of the time the server loaded the map, stats wouldnt save, because theres an explicit `StartupDelay` configured

**Comment by jweigelt** [comment]
When the server first starts, /status returns a empty map string playing mode "CON". As a workaround I added a bit of delay to prevent it from detecting map change for that particular map. @yonilerner
A better way of implementing this would probably be to explicitly check if the server is in it's default loading state.

I noticed that match statistics seem to log properly but individual player scores are not saved. This will mainly affect the !stats command but also the CSVs exported from web admin. Not sure if that is the same behaviour observed by @Legitosity?
